### PR TITLE
marked old content as deprecated

### PR DESCRIPTION
**Pull request by garethahealy**
@pabrahamsson 

cc: @redhat-cop/day-in-the-life

### CODE REVIEW

**Comment by springdo** [comment]
I have a funny feeling that the `ipa-server` one is used by something still Gareth, perhaps used for the Demo platform or for the old labs automation.... @eformat  / @ckavili  / @oybed might also know as well as Petter 🤷

**Comment by garethahealy** [comment]
@springdo reverted ipa move

**Review by pabrahamsson — DISMISSED** [review]
LGTM

**Comment by garethahealy** [comment]
reverted move and added to readme

File: OWNERS [hunk 1]
```diff
@@ -1,20 +0,0 @@
-# See the OWNERS docs at https://go.k8s.io/owners
```
Comment (oybed) [inline OWNERS:1]: Can we leave this file in place? ... alternatively update it if there are users that are no longer involved?

Comment (garethahealy) [reply OWNERS:1]: as we don't use prow anymore, this file doesn't do anything thats why I've replaced it with `CODEOWNERS` which uses the github configured teams.

Comment (oybed) [reply OWNERS:1]: Understood, but we also look at this file in a manual way, so by removing it we loose some of the valuable info. Is there a way to add a "default" to the `CODEOWNERS` file to keep the rest of the users from this file?

Comment (oybed) [reply OWNERS:1]: Cool - missed the "default" / * at the top of the `CODEOWNERS` file. There's a discrepancy between that group and the old file, but we can work on that separately.

**Pull request merged**